employees/templates/employees/partial/projects_work_percentage.html

```diff
@@ -0,0 +1,11 @@
+<h3>{{ UI_text.PROJECTS_WORK_PERCENTAGE_HEADER.value }}</h3>
+<table class="table">
+    <tbody>
+        {% for project, work_percentage in projects_work_percentage.items %}
+        <tr>
+            <td>{{ project }}</td>
+            <td>{{ work_percentage|floatformat:2 }} %</td>
+        </tr>
+        {% endfor %}
+    </tbody>
+</table>
```

Comment:
Why? Its not incorrect for HTML files I think. Please also note that this is partial, so each time it is included it will add blank line to the place where it is included.

Reply:
Ok, good point :+1: